core/computed/metrics/lcp-load-end.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license Copyright 2023 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {makeComputedArtifact} from '../computed-artifact.js';
+import {LCPLoadStart} from './lcp-load-start.js';
+
+class LCPLoadEnd extends LCPLoadStart {
+  /**
+   * @param {LH.Artifacts.NetworkRequest} record
+   * @return {number}
+   */
+  static getTimestamp(record) {
+    return record.networkEndTime * 1000;
+  }
```

Comment:
Definitely cheating here as `LCPLoadEnd extends LCPLoadStart` doesn't really make sense conceptually. This seemed like the easiest way to prevent code duplication.